package.json

```diff
@@ -29,9 +29,10 @@
     "start": "nf start",
     "start:dev": "NODE_ENV=development COMBINED=true babel-watch -L server",
     "ngrok": "ngrok http 8082 --region eu --subdomain sql",
-    "test": "npm run test:lint && npm run test:flow && npm run test:units",
+    "test": "npm run test:lint && npm run test:flow && npm run test:unit && npm run test:integration",
     "test:lint": "eslint server",
-    "test:units": "NODE_ENV=test mocha --require babel-register -R spec ./tests",
+    "test:integration": "NODE_ENV=test mocha --require babel-register -R spec ./tests/integration",
+    "test:unit": "NODE_ENV=test mocha --require babel-register -R spec ./tests/unit",
```

Comment:
You're right, thanks for pointing that out. There is one thing that is unique for these test directory and it is `test/support` directory where i moved filesystem adapter. I can't really move it to `unit` or `integration` because they both use it